pycbc/conversions.py

```diff
@@ -1119,12 +1119,17 @@ def freqlmn_from_other_lmn(f0, tau, current_l, current_m, new_l, new_m):
         correspond to an unphysical Kerr black hole mass and/or spin,
         ``numpy.nan`` will be returned.
     """
-    mass = final_mass_from_f0_tau(f0, tau, l=current_l, m=current_m)
-    spin = final_spin_from_f0_tau(f0, tau, l=current_l, m=current_m)
+    mass = numpy.array(final_mass_from_f0_tau(f0, tau, l=current_l,
+                                              m=current_m))
+    spin = numpy.array(final_spin_from_f0_tau(f0, tau, l=current_l,
+                                              m=current_m))
 
     mass[mass < 0] = numpy.nan
     spin[numpy.abs(spin) > 0.9996] = numpy.nan
 
+    if type(f0) is float and type(tau) is float:
```

Comment:
What you could do here instead is `if len(mass) == 1`, that would solve the issue with the ints. And we should not add in this loop `and len(spin) == 1`, we should  just raise an error above if mass and spin don't have the same lengt. 
But I am not quite convinced about this, is it that important to not return an array if the input is not an array?

Reply:
Maybe it is not an issue - but after the problems we had with the dimensions of arrays in the past, I was worried this might be an issue, too. I expect that depends on the functions using the output of the conversion.

Reply:
I just realized, one would hardly get mass and spin with different lengths, what the error should be raised on is if f0 and tau do not have the same length